voltdbclient/url.go

```diff
@@ -25,6 +25,7 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
+		// need go version >= 1.8 to use this method.
```

Comment:
@gernest By the way , i can update the version requirement if you write down your client go version support.